src/multirust-mock/src/clitools.rs

```diff
@@ -301,12 +299,27 @@ fn build_mock_channel(channel: &str, date: &str,
     }
 }
 
-pub fn this_host_triple(channel: &str) -> String {
-    ToolchainDesc::from_str(channel).ok().map(|t| t.target_triple()).unwrap()
+pub fn this_host_triple() -> String {
+    let arch = if cfg!(target_arch = "x86") { "i686" }
+    else if cfg!(target_arch = "x86_64") { "x86_64" }
+    else { unimplemented!() };
+    let os = if cfg!(target_os = "linux") { "unknown-linux" }
+    else if cfg!(target_os = "windows") { "windows" }
+    else if cfg!(target_os = "macos") { "macos" }
```

Comment:
I may have just randomly come across this by accident, but shouldn't this be `apple-darwin` and the Windows one be `pc-windows`?